CHANGELOG.md

```diff
@@ -268,6 +268,77 @@
 
   ([PR #1376](https://github.com/alphagov/govuk-frontend/pull/1376))
 
+- Ensure GOV.UK Frontend component selectors cannot conflict when initialised
+
+  Components which have JavaScript functionality include a data attribute called `[data-module]`,
+  we have added the `govuk-` prefix as a namespace.
+
+  For example the accordion component selector has changed from `[data-module="accordion"]` to `[data-module="govuk-accordion"]`.
+
+  ```html
+  <div class="govuk-accordion" data-module="govuk-accordion">
+    <!-- ... -->
+  </div>
+  ```
+
+  This ensures the component's name does not conflict with other service level components
+  with the same name, which is consistent with our CSS class name convention.
+
+  If you're using Nunjucks macros, and are [using the `initAll` function](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-1-include-javascript) you will not be affected.
+
+  If you are using HTML, you will need to add the `govuk-` prefix to any `[data-module]` attributes referencing GOV.UK Frontend components.
```

Comment:
I wonder if 
> referencing GOV.UK Frontend components.

is clear? Could we say 
> any `[data-module]` attributes on GOV.UK Frontend component HTML

We could add
> and any `[data-module]` attributes referencing GOV.UK Frontend scripts in your custom components

although it seems like a fairly unlikely scenario.

Reply:
Or alternatively
> any `[data-module]` attributes in HTML which reference GOV.UK Frontend Javascript 
🤷‍♀

Reply:
Will try and make this clearer thanks Hanna :)

Reply:
I've moved the example around and adjusted the wording so it hopefully makes it clearer, lemme know what you think.